### PR TITLE
docs: add chinese font recommend to other lang readme & fix readme link

### PR DESCRIPTION
**Pull request by littlecxm**
extend from [PR #406](https://github.com/nostalgic-css/NES.css/pull/406)

### CODE REVIEW

File: .github/README-pt-BR.md [hunk 1]
```diff
@@ -47,9 +47,10 @@ NES.css não fornece nenhuma fonte, mas nós mantemos uma lista de fontes recome
 | -------- | ------------------------------------------------------------------ |
 | (Padrão) | [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P) |
 | Inglês   | [Kongtext](https://www.dafont.com/kongtext.font)                   |
+| Chinoés  | [Zpix (最像素)](https://github.com/SolidZORO/zpix-pixel-font)      |
```
Comment (guastallaigor) [inline .github/README-pt-BR.md]: Actually it's "Chinês" here, could you please change it?

**Review by guastallaigor — CHANGES_REQUESTED** [review]
LGTM 👍 
Just one change in the pt-BR README.md file and I'll merge it.
Thanks again 🎉

File: .github/README-pt-BR.md (hunk 1 above)
Comment (littlecxm) [reply .github/README-pt-BR.md]: okay, got it :)

**Review by guastallaigor — APPROVED** [review]
LGTM 👍

**Pull request merged**